packages/genesys-spark-components/src/components/stable/gux-listbox-multi/gux-listbox-multi.tsx

```diff
@@ -70,6 +70,9 @@ export class GuxListboxMulti {
   @Prop()
   emptyMessage: string;
 
+  @Prop()
+  disabled: boolean = false;
```

Comment:
add a comment that this is used by child elements

Reply:
Fixed